codechecker/CodeChecker

```diff
@@ -66,9 +66,23 @@ def main():
     checker_env = original_env
 
     tmp_dir = tempfile.mkdtemp()
+    global is_tmp_removed
```

Comment:
Why do we need a global variable? Shouldn't be enough to check if `tmp_dir` still exists and remove it if it is there in `__remove_tmp()`?
I'm not a big fan of global variables.